code/game/machinery/doors/airlock.dm

```diff
@@ -1081,11 +1081,11 @@
 		charge.ex_act(EXPLODE_DEVASTATE)
```

Comment:
It should be EXPLODE_DEMOLISH, the word devestate could trigger potential vietnam PTSD when we dropped a bunch of napalm on those chings in that village. My god... the children are still screaming in my dreams.